config/locales/spotlight.en.yml

```diff
@@ -710,8 +710,9 @@ en:
         header: Metadata
         instructions: Select metadata fields to display on each type of page. Select a field name to edit its display label. Drag and drop fields to specify the order in which they are displayed.
         order_header: Display and order metadata fields
-        select_all: Select all
+        select_check: All
```

Comment:
Should we leave the translation key the same in case anyone has customized it?

Reply:
I put it back in.

Reply:
Sorry, I didn't mean keep both. I mean continue to use the existing select_all translation and set the value to "All" for display.

Reply:
ok, will make that change.